src/commands/go-lint.yaml

```diff
@@ -1,64 +1,15 @@
 parameters:
-  # Many additional parameters are supported by golanglint-ci.
-  disable:
-    description: "The names of default linting modules to skip."
-    type: "string"
-  disable-all:
-    description: "If set, will disable all linters (except those included by '--enable')."
-    type: "boolean"
-  enable:
-    description: "The names of additional linting modules to run."
-    type: "string"
   new-from-rev:
     description: "Show only new issues created after git revision REV."
     type: "string"
```

Comment:
Why would we keep that option open? Is the linter too slow to check all files every single time?

Reply:
No the speed is fine, the idea was just to provide some way to scope the check. For example in the case of a feature branch, we might be interested in only blocking a build for new issues. This could be removed, then we'll just always get output for the whole history

Reply:
I think we should make the decision here and not in every single repo build config. For me either is fine.

Reply:
Fine, I'd rather have everything visible in that case. Removed